hydra-node/test/Hydra/BehaviorSpec.hs

```diff
@@ -1,4 +1,5 @@
 {-# LANGUAGE DuplicateRecordFields #-}
+{-# LANGUAGE PatternSynonyms #-}
```

Comment:
question: This addition seems unrelated?

Reply:
Yes. This is from one commit I reverted, but somehow this persisted.

Reply:
https://github.com/input-output-hk/hydra-poc/compare/ac03f207ea97ebef138cd199e7e98f6264f12995..f71cc5c752e2b35163482d965e6e5b3e4f7a5acb